### ISSUE TITLE
ERR_HTTP2_PROTOCOL_ERROR on Google Chrome

### ISSUE THREAD

**Issue by virtazp** [glpi-project/glpi#14381]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.6

### Bug description

I have an error ERR_HTTP2_PROTOCOL_ERROR on Google Chrome if I do this action:

1. Create a ticket
2. I am typing a very long text (Ex: Lorem Ipsum 20 paragraphs)
3. I categorize the ticket

The error ERR_HTTP2_PROTOCOL_ERROR appears.

This bug does not occur on Firefox.

### Relevant log output

```shell
No relevant errors in the logs
```

### Page URL

https://mydomain/front/ticket.form.php

### Steps To reproduce

1. Create a ticket
2. I am typing a very long text (Ex: Lorem Ipsum 20 paragraphs)
3. I categorize the ticket

### Your GLPI setup information

<details><summary>Informations sur le système, l'installation et la configuration</summary><pre>GLPI 10.0.6 ( => /var/www/xxxxx)
Installation mode: TARBALL
Current language:fr_FR

</details><details><summary>Server</summary><pre> 
Operating system: Linux xxxxx 5.10.0-19-amd64 #1 SMP Debian 5.10.149-2 (2022-10-21) x86_64
PHP 7.4.33 fpm-fcgi (Core, FFI, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, bz2, calendar, cgi-fcgi, ctype, curl, date,
	dom, exif, fileinfo, filter, ftp, gd, gettext, gmp, hash, iconv, imap, intl, json, libxml, mbstring, mysqli, mysqlnd, openssl,
	pcre, pdo_mysql, posix, readline, session, shmop, soap, sockets, sodium, standard, sysvmsg, sysvsem, sysvshm, tokenizer, xml,
	xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="8M" 
Software: nginx/1.18.0
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36
Server Software: Debian 11
	Server Version: 10.5.18-MariaDB-0+deb11u1
	Server SQL Mode: STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
	Parameters: xxxxx
	Host info: 127.0.0.1 via TCP/IP
	
PHP version (7.4.33) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.
gd extension is installed.
intl extension is installed.
libxml extension is installed.
zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (10.5.18) is supported.
No files from previous GLPI version detected.
The log file has been created successfully.
Write access to /var/www/xxxxx/files/_cache has been validated.
Write access to /var/www/xxxxx/config has been validated.
Write access to /var/www/xxxxx/files/_cron has been validated.
Write access to /var/www/xxxxx/files has been validated.
Write access to /var/www/xxxxx/files/_dumps has been validated.
Write access to /var/www/xxxxx/files/_graphs has been validated.
Write access to /var/www/xxxxx/files/_lock has been validated.
Write access to /var/www/xxxxx/files/_pictures has been validated.
Write access to /var/www/xxxxx/files/_plugins has been validated.
Write access to /var/www/xxxxx/files/_rss has been validated.
Write access to /var/www/xxxxx/files/_sessions has been validated.
Write access to /var/www/xxxxx/files/_tmp has been validated.
Write access to /var/www/xxxxx/files/_uploads has been validated.
The following directories should be placed outside "/var/www/xxxxx":
‣ "/var/www/xxxxx/files" ("GLPI_VAR_DIR")
‣ "/var/www/xxxxx/config" ("GLPI_CONFIG_DIR")
You can ignore this suggestion if you are certain that these directories are not accessible through your web server.
Sessions configuration is secured.
exif extension is installed.
ldap extension is not present.
openssl extension is installed.
zip extension is installed.
bz2 extension is installed.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /var/www/xxxxx/marketplace has been validated.
Access to timezone database (mysql) is not allowed.

</details><details><summary>GLPI constants</summary><pre> 
GLPI_ROOT: "/var/www/xxxxx"
GLPI_CONFIG_DIR: "/var/www/xxxxx/config"
GLPI_VAR_DIR: "/var/www/xxxxx/files"
GLPI_MARKETPLACE_DIR: "/var/www/xxxxx/marketplace"
GLPI_USE_CSRF_CHECK: "1"
GLPI_CSRF_EXPIRES: "7200"
GLPI_CSRF_MAX_TOKENS: "100"
GLPI_USE_IDOR_CHECK: "1"
GLPI_IDOR_EXPIRES: "7200"
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: false
GLPI_SERVERSIDE_URL_ALLOWLIST: ["/^(https?|feed):\\/\\/[^@:]+(\\/.*)?$/"]
GLPI_TELEMETRY_URI: "https://telemetry.glpi-project.org"
GLPI_INSTALL_MODE: "TARBALL"
GLPI_NETWORK_MAIL: "glpi@teclib.com"
GLPI_NETWORK_SERVICES: "https://services.glpi-network.com"
GLPI_MARKETPLACE_ALLOW_OVERRIDE: true
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: true
GLPI_USER_AGENT_EXTRA_COMMENTS: ""
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: "1"
GLPI_AJAX_DASHBOARD: "1"
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: "0"
GLPI_CENTRAL_WARNINGS: "1"
GLPI_DOC_DIR: "/var/www/xxxxx/files"
GLPI_CACHE_DIR: "/var/www/xxxxx/files/_cache"
GLPI_CRON_DIR: "/var/www/xxxxx/files/_cron"
GLPI_DUMP_DIR: "/var/www/xxxxx/files/_dumps"
GLPI_GRAPH_DIR: "/var/www/xxxxx/files/_graphs"
GLPI_LOCAL_I18N_DIR: "/var/www/xxxxx/files/_locales"
GLPI_LOCK_DIR: "/var/www/xxxxx/files/_lock"
GLPI_LOG_DIR: "/var/www/xxxxx/files/_log"
GLPI_PICTURE_DIR: "/var/www/xxxxx/files/_pictures"
GLPI_PLUGIN_DOC_DIR: "/var/www/xxxxx/files/_plugins"
GLPI_RSS_DIR: "/var/www/xxxxx/files/_rss"
GLPI_SESSION_DIR: "/var/www/xxxxx/files/_sessions"
GLPI_TMP_DIR: "/var/www/xxxxx/files/_tmp"
GLPI_UPLOAD_DIR: "/var/www/xxxxx/files/_uploads"
GLPI_INVENTORY_DIR: "/var/www/xxxxx/files/_inventories"
GLPI_NETWORK_REGISTRATION_API_URL: "https://services.glpi-network.com/api/registration/"
GLPI_MARKETPLACE_PLUGINS_API_URI: "https://services.glpi-network.com/api/glpi-plugins/"
GLPI_I18N_DIR: "/var/www/xxxxx/locales"
GLPI_VERSION: "10.0.6"
GLPI_SCHEMA_VERSION: "10.0.6@21cffee0fbb5afbf0d580cabdf6fd7a922598f97"
GLPI_MARKETPLACE_PRERELEASES: false
GLPI_MIN_PHP: "7.4.0"
GLPI_MAX_PHP: "8.3.0"
GLPI_YEAR: "2023"

</details><details><summary>Libraries</summary><pre> 
htmlawed/htmlawed version 1.2.9 in (/var/www/xxxxx/vendor/htmlawed/htmlawed)
phpmailer/phpmailer version 6.6.0 in (/var/www/xxxxx/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.8 in (/var/www/xxxxx/vendor/simplepie/simplepie/library)
tecnickcom/tcpdf version 6.6.2 in (/var/www/xxxxx/vendor/tecnickcom/tcpdf)
michelf/php-markdown in (/var/www/xxxxx/vendor/michelf/php-markdown/Michelf)
true/punycode in (/var/www/xxxxx/vendor/true/punycode/src)
iamcal/lib_autolink in (/var/www/xxxxx/vendor/iamcal/lib_autolink)
sabre/dav in (/var/www/xxxxx/vendor/sabre/dav/lib/DAV)
sabre/http in (/var/www/xxxxx/vendor/sabre/http/lib)
sabre/uri in (/var/www/xxxxx/vendor/sabre/uri/lib)
sabre/vobject in (/var/www/xxxxx/vendor/sabre/vobject/lib)
laminas/laminas-i18n in (/var/www/xxxxx/vendor/laminas/laminas-i18n/src)
laminas/laminas-servicemanager in (/var/www/xxxxx/vendor/laminas/laminas-servicemanager/src)
monolog/monolog in (/var/www/xxxxx/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/var/www/xxxxx/vendor/sebastian/diff/src)
donatj/phpuseragentparser in (/var/www/xxxxx/vendor/donatj/phpuseragentparser/src/UserAgent)
elvanto/litemoji in (/var/www/xxxxx/vendor/elvanto/litemoji/src)
symfony/console in (/var/www/xxxxx/vendor/symfony/console)
scssphp/scssphp in (/var/www/xxxxx/vendor/scssphp/scssphp/src)
laminas/laminas-mail in (/var/www/xxxxx/vendor/laminas/laminas-mail/src/Protocol)
laminas/laminas-mime in (/var/www/xxxxx/vendor/laminas/laminas-mime/src)
rlanvin/php-rrule in (/var/www/xxxxx/vendor/rlanvin/php-rrule/src)
blueimp/jquery-file-upload in (/var/www/xxxxx/vendor/blueimp/jquery-file-upload/server/php)
ramsey/uuid in (/var/www/xxxxx/vendor/ramsey/uuid/src)
psr/log in (/var/www/xxxxx/vendor/psr/log/Psr/Log)
psr/simple-cache in (/var/www/xxxxx/vendor/psr/simple-cache/src)
psr/cache in (/var/www/xxxxx/vendor/psr/cache/src)
league/csv in (/var/www/xxxxx/vendor/league/csv/src)
mexitek/phpcolors in (/var/www/xxxxx/vendor/mexitek/phpcolors/src/Mexitek/PHPColors)
guzzlehttp/guzzle in (/var/www/xxxxx/vendor/guzzlehttp/guzzle/src)
guzzlehttp/psr7 in (/var/www/xxxxx/vendor/guzzlehttp/psr7/src)
glpi-project/inventory_format in (/var/www/xxxxx/vendor/glpi-project/inventory_format/lib/php)
wapmorgan/unified-archive in (/var/www/xxxxx/vendor/wapmorgan/unified-archive/src)
paragonie/sodium_compat in (/var/www/xxxxx/vendor/paragonie/sodium_compat/src)
symfony/cache in (/var/www/xxxxx/vendor/symfony/cache)
html2text/html2text in (/var/www/xxxxx/vendor/html2text/html2text/src)
symfony/css-selector in (/var/www/xxxxx/vendor/symfony/css-selector)
symfony/dom-crawler in (/var/www/xxxxx/vendor/symfony/dom-crawler)
twig/twig in (/var/www/xxxxx/vendor/twig/twig/src)
twig/string-extra in (/var/www/xxxxx/vendor/twig/string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 in (/var/www/xxxxx/vendor/symfony/polyfill-php80)
symfony/polyfill-php81 in (/var/www/xxxxx/vendor/symfony/polyfill-php81)
symfony/polyfill-php82 in (/var/www/xxxxx/vendor/symfony/polyfill-php82)

</details><details><summary>SQL replicas</summary><pre> 
Not active

</details><details><summary>Notifications</summary><pre> 
Way of sending emails: SMTP+TLS (xxxxx)

</details><details><summary>Plugins list</summary><pre> 
	fields               Name: Champs supplémentaires         Version: 1.18.1     State: Enabled                                 
		Install Method: Manual
	behaviors            Name: Comportements                  Version: 2.7.1      State: Enabled                                 
		Install Method: Manual
	fusioninventory      Name: FusionInventory                Version: 10.0.1+1.0 State: Enabled                                 
		Install Method: Manual
	customnotifications  Name: Mail ticket urgency            Version: 1.0.0      State: Enabled                                 
		Install Method: Manual
	singlesignon         Name: Single Sign-on                 Version: 1.3.3      State: Enabled                                 
		Install Method: Manual

</pre></details>

**Comment by cconard96** [comment]
If you are sending an extreme amount of data to GLPI, you are likely hitting the `post_max_size` limit set for PHP which seems to be set at the default 8M currently. When you change categories, the current form data is sent to GLPI to be saved, and the page is reloaded to recalculate the template being used. Please increase this limit on your server, and test again. After changing the limit in the PHP config, you will need to restart your web server for the change to take effect.

**Comment by virtazp** [comment]
Good morning,

I already had this condition in my nginx http block: client_max_body_size 200M;

Following your comment, I modified php.ini:
upload_max_filesize = 64M
&
post_max_size = 64M

I restarted php7.4-fpm and nginx service.

The problem is still here. I have no errors in the GLPI logs and in the Nginx or php logs.

I don't think there is a problem with my configuration because sometimes it works. Also, it works all the time on Firefox. The problem is present only on Google Chrome.
